docs/src/variables.md

```diff
@@ -44,6 +44,10 @@ This code does three things:
 To reduce confusion, we will attempt, where possible, to always refer to
 variables with their corresponding prefix.
 
+!!! warn
+    If you create two JuMP variables with the same name, an error will be
```

Comment:
More directly: "Creating two JuMP variables with the same name results in an error at runtime."